src/scenes/home/header/sideNav/sideNav.css

```diff
@@ -40,7 +40,7 @@
   margin-top: 80px;
   padding: 30px 5%;
   border-top: 1px solid #fafafa;
-  font-size: 2.5em;
+  font-size: xx-large;
```

Comment:
Can you explain why we should use this over the hard-set number?

This applies to the topNav.css change as well.